2019-09-17 21:43:59,285 ERROR: Exception on /reset_password_request [GET] [in d:\myproject\education_proj\app\flask\education_flask\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\myproject\education_proj\app\flask\education_flask\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\myproject\education_proj\app\flask\education_flask\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "d:\myproject\education_proj\app\flask\education_flask\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "d:\myproject\education_proj\app\flask\education_flask\lib\site-packages\flask\app.py", line 2097, in make_response
    "The view function did not return a valid response. The"
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
